static/js/containers/AuthRequiredPage.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+/* global SETTINGS: false */
+import React from "react"
+import DocumentTitle from "react-document-title"
+
+import Card from "../components/Card"
+import { formatTitle } from "../lib/title"
```

Comment:
can you put a blank line in between the component import and the function import?